.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,59 @@
+name: Bug report
+description: Scanpy doesn’t do what it should? Please help us fix it!
+#title: ...
+labels: [bug]
+#assignees: []
+body:
+- type: checkboxes
+  id: terms
+  attributes:
+    label: Please make sure these conditions are met
+    # description: ...
+    options:
+    - label: I have checked that this issue has not already been reported.
+      required: true
+    - label: I have confirmed this bug exists on the latest version of scanpy.
+      required: true
+    - label: (optional) I have confirmed this bug exists on the master branch of scanpy.
+      required: false
+- type: markdown
+  attributes:
+    value: |
+      **Note**: Please read [this guide](https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports) detailing how to provide the necessary information for us to reproduce your bug.
+- type: textarea
+  id: description
+  attributes:
+    label: What happened?
+    description: Describe what you tried to do, and what happened instead.
+  validations:
+    required: true
+- type: textarea
+  id: code
+  attributes:
+    label: Minimal code sample
+    description: (that we can copy&paste without having any data)
+    render: python
+  validations:
+    required: true
+- type: textarea
+  id: traceback
+  attributes:
+    label: Error output
+    description: Produced by the code above. Leave empty if the issue is unexpected behavior instead of an error.
+    render: pytb
+  validations:
+    required: false
+- type: textarea
+  id: versions
+  attributes:
+    label: Versions
+    description: Paste the output of scanpy.logging.print_versions() leaving a blank line after the details tag
+    render: markdown
+    value: |
+      <details>
```

Comment:
Think we could add a placeholder here

Reply:
A placeholder is the grey text that vanishes once you focus the field, `value` stays.

Since there’s no way to express “add `<details>` before the user content and `</details>` after”, this is the best we can do